popup.html

```diff
@@ -1,51 +1,75 @@
-  </div>
-  <div id="user_box">
-    <div id="profile_box">
-      <!-- Profile info area -->
+
+<!-- Spinner to display when loading -->
+<div id="loader">
+    <div class="centered-content">
+        <img src="images/loading.gif" alt="Loading" id="loading-gif">
     </div>
-    <div id="loggedin">
-      <button id="logout_btn">Logout</button>
+</div>
+
+<!-- The 'Login' button appears when not logged in on Channel-i -->
+<div id="login">
+    <div class="centered-content">
+        <img src="images/img_logo.png" alt="IMG" class="ui image" id="img-logo-img">
+        <div class="ui labeled icon button" id="login-button">
+            <i class="sign in icon"></i>Log in
+        </div>
     </div>
-  </div>
-  <div id="login">
-    <form id="login_form" method="POST">
-      <input type="text" class="field" id="username_field" name="username" value="" placeholder="Username" required/>
-      <input type="password" class="field" id="password_field" name="password" value="" placeholder="Password" required/>
-      <input type="submit" id="submit" value="">
-    </form>
-  </div>
-  <div id="main">
-    <!-- Main apps box -->
-  </div>
-
-  <a href="https://www.facebook.com/IMGIITRoorkee" target="_blank" id="footer">
-    <img style="position: absolute;right: 14px;bottom: 10px;width:40px;height: 	40px;" src="images/img_logo.png">
-  </a>
-
-  <script type="text/javascript" src="js/jquery-1.9.1.js"></script>
-  <script type="text/javascript" src="js/popup.js"></script>
+</div>
+
+<!-- The profile and links appear when logged in on Channel-i-->
+<div id="main">
+    <div class="ui fluid card">
+        <div class="content">
+            <img src="" alt="" class="ui right floated circular image" id="user-photo">
+            <div class="header" id="user-name"></div>
+            <div class="meta" id="user-username"></div>
+        </div>
+        <div class="extra content">
+            <div class="ui labeled icon negative button" id="logout-button">
+                <i class="sign out icon"></i>Log out
+            </div>
+        </div>
+    </div>
+    <table class="ui unstackable celled table">
+        <tbody id="links-table-body">
+        </tbody>
+    </table>
+    <div class="ui divider"></div>
+    <a href="https://channeli.in/">
+        <img src="images/channel-i-logo.png" alt="Channel-i" class="ui middle aligned image" id="channel-i-logo">
+    </a>
+    &nbsp;&nbsp;by&nbsp;&nbsp;
+    <a href="https://www.facebook.com/IMGIITRoorkee/">
+        <img src="images/img_logo.png" alt="IMG" class="ui middle aligned image" id="img-logo-footer">
+    </a>
+    <p></p>
+    <p>It's everything you need it to be.</p>
+</div>
 </body>
 
+<!-- jQuery -->
+<script type="text/javascript" src="js/jquery.js"></script>
+<!-- Semantic UI JS -->
+<script type="text/javascript" src="js/semantic.js"></script>
+<!-- Custom JS -->
+<script type="text/javascript" src="js/popup.js"></script>
+<script type="text/javascript" src="js/apps.js"></script>
+
 </html>
```

Comment:
To be honest, it was greedy... I wanted to be an organization member getting my pull request merged was the only viable option :wink: